sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-full-time-feedback-integration.scheduler.e2e-spec.ts

```diff
@@ -385,5 +395,83 @@ describe(
         });
       },
     );
+
+    it("Should generate a notification to the ministry when there are ecert feedback errors that block funding for a disbursement.", async () => {
+      // Arrange
+      const notificationMessageType =
+        NotificationMessageType.ECertFeedbackFileErrorNotification;
+      // Remove all the eCert feedback file error notification records before
+      // the run of the test, so that it doesn't interfere with the test data.
+      await db.notification.delete({
```

Comment:
why we are deleting the notifications? does it interfere with the test data?

Reply:
I believe that most of the time we update the notification, for instance, with a dummy date sent instead of deleting it. I would recommend to follow it.
https://github.com/bcgov/SIMS/blob/96dcee9843d9d35b2561674b1b5eccb48b5264e6/sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-full-time-process-response-integration.scheduler.e2e-spec.ts#L78-L81

Reply:
There are 2 notification tests in 2 different files. Therefore, I have added the `update` db query in the e2e test itself.